archlinuxcn/wine-wechat/wine-wechat.desktop

```diff
@@ -11,3 +11,19 @@ Terminal=0
 Type=Application
 Categories=Network;InstantMessaging;
 Keywords=wx;wechat;weixin
+StartupWMClass=wechat
+Actions=Configure;Directory;
+
+[Desktop Action Configure]
+Exec=env WINEDEBUG=-all wechat --config
+Name=Configure Wine
+Name[zh_CN]=Wine 微信配置
+Comment=Run winecfg for Wine WeChat
+Comment[zh_CN]=为 Wine 微信运行 winecfg
+
+[Desktop Action Directory]
+Exec=env WINEDEBUG=-all wechat --dir
+Name=User Directory
```

Comment:
ditto.

Reply:
ditto; please suggest how to phrase this accurately & concisely.

I've allowed edits by maintainers, so please feel free to change it as you like; I might not be able to respond in time...